backend/src/main/java/com/benine/backend/http/HttpController.java

```diff
@@ -67,7 +67,7 @@ private void createHandlers() {
    * Creates the handlers for a certain camera.
    * @param cam camera to create handlers for.
    */
-  private void createHandlers(Camera cam) {
+  public void createHandlers(Camera cam) {
```

Comment:
If you changed this to public just for testing purposes I think you should create a mock object and use that for testing, that way you can access private methods.